File: TaskManager/Unit_Test/src/AddCommentTest.java
import org.junit.jupiter.api.Test;

public class AddCommentTest {

    @Test
    void addComment_ShouldAddCommentToIssue() {
        repository.addIssue(testIssue);
        Comment comment = new Comment("This is urgent!", "John");

        testIssue.addComment(comment);

        assertEquals(1, testIssue.comments.size());
        assertEquals("This is urgent!", testIssue.comments.get(0).content);
    }
}

File: TaskManager/TrelloApp/TrelloApp.Test/AddCommentTest.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace TrelloApp.Test
{
    public class AddCommentTest
    {
        [Fact]
        public void AddComment_ShouldAddCommentToIssue()
        {
            int id = 1;
            Comment comment = new Comment(id, "Content nai", "Labib", 21);
            Issue issue = new Issue();

            issue.addComment(comment);

            var commentInfo = issue.comments[id-1];

            Assert.Equal(id, commentInfo.id);
            Assert.Equal(comment.content, commentInfo.content);
            Assert.Equal(comment.author, commentInfo.author);
            Assert.Equal(comment.createdAt, commentInfo.createdAt);

        }
    }
}
